webdriver/tests/bidi/script/call_function/arguments.py

```diff
@@ -0,0 +1,116 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "argument, expected",
+    [
+        ({"type": "undefined"}, "undefined"),
+        ({"type": "null"}, "null"),
+        ({"type": "string", "value": "foobar"}, "'foobar'"),
+        ({"type": "string", "value": "2"}, "'2'"),
+        ({"type": "number", "value": "-0"}, "-0"),
+        ({"type": "number", "value": "Infinity"}, "Infinity"),
+        ({"type": "number", "value": "-Infinity"}, "-Infinity"),
+        ({"type": "number", "value": 3}, "3"),
+        ({"type": "number", "value": 1.4}, "1.4"),
+        ({"type": "boolean", "value": True}, "true"),
+        ({"type": "boolean", "value": False}, "false"),
+        ({"type": "bigint", "value": "42"}, "42n"),
+    ],
+)
+async def test_primitive_values(bidi_session, top_context, argument, expected):
+    result = await bidi_session.script.call_function(
+        function_declaration=
+        f"(arg) => {{ "
+        f"    if(arg!=={expected}) "
+        f"        throw Error(\"Argument type should be {expected}, but was \"+arg);"
+        f"    return arg;"
```

Comment:
Let's just return `arg===expected` and check that `result` is `True`?

Reply:
Having an exception with a meaningful message thrown makes debugging way easier, as the error message contains the meaningful message, rather than "expected `true`, but was `false`".